package.json

```diff
@@ -40,11 +42,18 @@
   },
   "devDependencies": {
     "@changesets/cli": "^2.16.0",
+    "@rollup/plugin-commonjs": "^21.0.0",
+    "@rollup/plugin-node-resolve": "^13.0.5",
     "@types/prettier": "^2.4.1",
-    "ava": "^4.0.0-alpha.2",
+    "ava": "4.0.0-alpha.2",
```

Comment:
I'm currently having problems with the new version. I think we can solve this in another pr.